contracts/JBExpirySource.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import './interfaces/IJBExpirySource.sol';
+
+contract JBExpirySource is IJBExpirySource {
+  /** 
+    @notice
+    The number of seconds in 365 days.
+  */
+  uint256 private constant _SECONDS_IN_YEAR = 31536000;
+
+  /** 
+    @notice Get the total challenge expiry time when challenging an inactive project handle.
+
+    @dev The projectId param is unused here, but could be used in other implementations of IJBChallengeExpiry 
+    (e.g. to help recover specific handles)
+  */
+
+  function getExpiryFor(uint256) external view override returns (uint256) {
```

Comment:
`getChallengePeriod"?

maybe the `expiry` metaphor is clicking for you well, im happy to go with that instead of the `challengePeriod` metaphor, but just wanna make sure.

`challengePeriod` to me suggests the time the the owner of the handle has to answer the challenge by renewing.

Reply:
Down for either, but my inclination is either `expiry` everywhere or `period` everywhere. Should we do period?

Reply:
cool. `period` or `challengePeriod`?

leaning towards `challengePeriod` cus `expiry` and `JBExpiry` is a little non-descript.

Reply:
sounds good, will update it